pkg/backend/backend.go

```diff
@@ -222,3 +222,30 @@ func (c *backendClient) GetStackOutputs(ctx context.Context, name string) (resou
 	}
 	return res.Outputs, nil
 }
+
+// GetStackResourceOutputs returns the outputs of the stack with the given name.
+func (c *backendClient) GetStackResourceOutputs(
+	ctx context.Context, name, typ string) (resource.PropertyMap, error) {
+	ref, err := c.backend.ParseStackReference(name)
+	if err != nil {
+		return nil, err
+	}
+	s, err := c.backend.GetStack(ctx, ref)
+	if err != nil {
+		return nil, err
+	}
+	if s == nil {
+		return nil, errors.Errorf("unknown stack \"%s\"", name)
```

Comment:
Nit: You can use `%q` as a format specifier, which will accept a string and print it with quotation marks around it. (So you don't have to add the `\"` yourself.